polynote-kernel/src/main/scala/polynote/kernel/util/GlobalInfo.scala

```diff
@@ -0,0 +1,94 @@
+package polynote.kernel.util
+
+import java.io.File
+import java.net.URL
+
+import scala.reflect.internal.util.{AbstractFileClassLoader, BatchSourceFile}
+import scala.reflect.internal.util.ScalaClassLoader.URLClassLoader
+import scala.reflect.io.{AbstractFile, VirtualDirectory}
+import scala.tools.nsc.Settings
+import scala.tools.nsc.interactive.Global
+
+final case class GlobalInfo(global: Global, classPath: List[File], classLoader: AbstractFileClassLoader)
```

Comment:
And I guess this will be subsumed in some way (along with `RuntimeSymbolTable`) by `RuntimeContext`

Reply:
Right, the RuntimeContext will be the owner of Global and thus there won't be a need for this case class.